Automattic/portkey-android
# Add view binding for IntroActivity in demo app

Fixes an issue where a fresh install of the Loop app would crash on launch in the view binding branch. It looks like the view binding setup for `IntroActivity` was missing, probably nobody noticed because the activity isn't shown once you've already granted the permissions the app needs.

### To test

1. Uninstall the `loop` app if you have it installed
2. Build this branch (there should be an installable build comment made automatically on this PR)
3. Confirm that you see an intro screen, you're able to grant permissions, and are taken to the next screen (which says "Still working on this! Tap the + to continue.")
